Fix gofmt formatting violation in router/router.go

Running `make lint` with `gofmt` enabled reports a formatting violation:

```
router/router.go:40:1: File is not properly formatted (gofmt)
	MentionRoutes           map[string]MentionRoute
^
```

**To fix:** Run `gofmt -w router/router.go` (or `gofmt -w ./...`) and commit the result.

--- a/router/router.go
+++ b/router/router.go
@@ -37,15 +37,15 @@ type RegisteredRoute struct {
 
 // Router the HTTP router which handles Events from Slack
 type Router struct {
-	MentionRoutes           map[string]MentionRoute
-	ChannelMessageRoutes    map[string]ChannelMessageRoute
-	SlashCommandRoutes      map[string]SlashCommandRoute
-	DefaultMentionRoute     MentionRoute
-	DeniedMentionRoute          MentionRoute
-	DeniedChannelMessageRoute   ChannelMessageRoute
-	DeniedSlashCommandRoute     SlashCommandRoute
-	DbConnection            *gorm.DB
-	BotUID                  string
+	MentionRoutes             map[string]MentionRoute
+	ChannelMessageRoutes      map[string]ChannelMessageRoute
+	SlashCommandRoutes        map[string]SlashCommandRoute
+	DefaultMentionRoute       MentionRoute
+	DeniedMentionRoute        MentionRoute
+	DeniedChannelMessageRoute ChannelMessageRoute
+	DeniedSlashCommandRoute   SlashCommandRoute
+	DbConnection              *gorm.DB
+	BotUID                    string
 }
 
 // this is required because slack-go doesn't seem to provide a way to get the bot's own ID

